src/ui/connect-ui/src/FamilyHubs.Referral.Core/ApiClients/OrganisationClientService.cs

```diff
@@ -149,7 +149,7 @@ public async Task<List<KeyValuePair<TaxonomyDto, List<TaxonomyDto>>>> GetCategor
 
     private static string GetPositionUrl(string? serviceType, double? latitude, double? longitude, double? proximity, string status, int pageNumber, int pageSize)
     {
-        return $"api/services-simple?serviceType={serviceType}&status={status}&pageNumber={pageNumber}&pageSize={pageSize}&isFamilyHub=false{(
+        return $"api/services-simple?serviceType={serviceType}&status={status}&pageNumber={pageNumber}&pageSize={pageSize}{(
```

Comment:
Maybe a StringBuilder is a better option here as this looks really hard to follow what is going on building the URL up?

Reply:
We already had most of one, done